src/audiomixerboard.cpp

```diff
@@ -1349,29 +1349,37 @@ void CAudioMixerBoard::AutoAdjustAllFaderLevels()
 
```

Comment:
On the previous line... that `4` should really be `MAX_NUM_CHANNELS` shouldn't it?  (Plus correct the `{}` style...)